src/main/java/me/totalfreedom/totalfreedommod/config/ConfigEntry.java

```diff
@@ -41,6 +41,9 @@
     ADMINLIST_CLEAN_THESHOLD_HOURS(Integer.class, "adminlist.clean_threshold_hours"),
     ADMINLIST_CONSOLE_IS_SENIOR(Boolean.class, "adminlist.console_is_senior"),
     //
+    COREPROTECT_AUTO_WIPING_ENABLED(Boolean.class, "coreprotect.auto_wipe"),
+    COREPROTECT_FILE_LIMIT(Integer.class, "coreprotect.file_limit"),
+    //
```

Comment:
As previously noted, would also suggest adding an option (Maybe further up the config?) to set if CoreProtect will even be used or not on the server.